app/controllers/standup_meetings_controller.rb

```diff
@@ -1,6 +1,6 @@
 class StandupMeetingsController < ApplicationController
   def index
-    @meeting_date = params[:date].nil? ? Date.current : Date.parse(params[:date])
+    @meeting_date = params[:date].nil? ? Date.current : Date.strptime(params[:date], '%m-%d-%Y')
```

Comment:
need to `strptime` because we're using a US friendly date format, but a less ruby friendly format.